dev/io.openliberty.microprofile.telemetry.1.0.internal/bnd.bnd

```diff
@@ -51,17 +53,20 @@ Service-Component: \
 # Need an explicit import for api.common because it gets used by weld proxies
 # created for other API classes
 Import-Package: \
-	io.opentelemetry.api.common,\
-	io.openliberty.microprofile.telemetry.common.*,\
-	*
+  com.ibm.websphere.ras,\
+  com.ibm.websphere.ras.annotation,\
+  io.opentelemetry.api.common,\
+  io.openliberty.microprofile.telemetry.common.*,\
```

Comment:
I think this package should have been `io.openliberty.microprofile.telemetry.internal.common`